src/test/java/com/google/api/codegen/testdata/java_test_test_library.baseline

```diff
@@ -199,12 +267,19 @@ public class LibraryServiceTest {
   @Test
   @SuppressWarnings("all")
   public void listBooksTest() {
+    ListBooksResponse response = ListBooksResponse.newBuilder()
+      .build();
+    List<GeneratedMessage> responses = new ArrayList<>();
```

Comment:
As discussed offline, I would fix this in a separate PR.
Issue to track: https://github.com/googleapis/toolkit/issues/354